app/commands/track/maintainership/create.rb

```diff
@@ -0,0 +1,13 @@
+class Track::Maintainership
+  class Create
```

Comment:
I've only created this command, and have not done anything with updating/removing maintainership. Want me to add those?